src/Domain/Services/Email/CustomerNewAccout.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * Class CustomerNewAccount file.
+ *
+ * @package  WooCommerce/Blocks
+ */
+
+namespace Automattic\WooCommerce\Blocks\Domain\Services\Email;
+
+use \WP_User;
+use \WC_Email;
+use Automattic\WooCommerce\Blocks\Domain\Package;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
```

Comment:
Do we actually need this? The file doesn't execute anything if loaded directly and at most will just fatal error because `\WC_Email` won't be available? I know this is a common pattern in WP but I've always been puzzled by it's blanket use.

Reply:
Good point! This is a remnant from paste-adapt of woo core code. Fixed here: f5cd2d5f6ecaf08e77882ab9e0b2d16df50df67c